modules/raft/src/main/java/org/apache/ignite/raft/jraft/rpc/ActionRequest.java

```diff
@@ -32,9 +34,15 @@ public interface ActionRequest extends Message {
     String groupId();
 
     /**
-     * @return Action's command.
+     * @return Serialized action's command. Specific serialization format may differ from group to group.
+     */
+    byte[] command();
```

Comment:
Question to myself. Maybe we should have a setter for this property as well. See `patchCommandBeforeApply`